src/paginate.ts

```diff
@@ -90,17 +85,12 @@ export class Paginate<Model extends PrismaClientModel> {
         ? this.paginationArgs.pageIndex + 1
         : 1;
     const hasNextPage = page < totalPages;
-    const hasPrevPage =
-      count > 0 && page > 1
-        ? (page * this.paginationArgs.limit) / count - 1 === 0 ||
-          page - 1 === totalPages
-        : false;
```

Comment:
`hasPrevPage` intend to validate if the previous page has data, on this change its value will be `true` if `page` exceed max of pages.

Reply:
I will update this logic to factor this in but it's worrying that my change didn't cause any of the other tests to fail